ur5_moveit_config/config/ur5.srdf

```diff
@@ -10,10 +10,10 @@
     <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
     <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
     <group name="manipulator">
-        <chain base_link="base_link" tip_link="ee_link" />
+        <chain base_link="base_link" tip_link="flange" />
```

Comment:
According to the [ROS-I](https://wiki.ros.org/Industrial/Tutorials/Create%20a%20URDF%20for%20an%20Industrial%20Robot#Additional.2FStandard_Frames) definition, I think this should be "tool0". But I am not sure...

Reply:
I agree with changing this to `tool0`

Reply:
https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/14/commits/a6e31bcafcf9317f341249e47985d8776197e97b